docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3242,21 +3263,32 @@ would be listed in MM/DD/YYYY format as 08/12/2022).
 **Incorrect Select** 
 
 Incorrect select applies to a range of data types. For this, select the sample to 
-have noise added. For those selected, randomly select a different option from the 
-correct one. This is randomly chosen from the list of options in `this csv <https://github.com/ihmeuw/vivarium_research_prl/blob/main/noise/incorrect_select_options.csv>`_. Note that for relationship to head of household, this includes 
-the full list of options, not just those seen in the household. 
+have noise added. For those selected, randomly select a new option. This is chosen 
+from the list of options in `this csv <https://github.com/ihmeuw/vivarium_research_prl/blob/main/src/vivarium_research_prl/noise/incorrect_select_options.csv>`_. Note that for relationship to head of household, this includes the full list of options, not just those seen in the household. 
 
 Limitations: 
 
 - For single person homes, incorrectly selecting relationship to head of household does not make as much sense. However, we continue with it here anyways. 
 - Incorrect selection likely takes place in a logical way, and might persist across observers (e.g., trans or nonbinary people "incorrectly" selecting a sex; confusion with different race/ethnicity groups; selecting a state from a prior address) however, we are not including this complexity. 
+- The current version of the function does not enforce that the new selection is different than the original selection. This means that some rows designated for incorrect selection will not actually be incorrect. This is especially important for sex as there are only 2 options to select. 
```

Comment:
What does this refer to? The version in the vivarium_research_prl repo? Having this as a limitation is a bit confusing, if we plan to implement it differently for the real package.